src/devices/Ws28xx/BitmapImageNeo3.cs

```diff
@@ -38,7 +36,12 @@ public override void SetPixel(int x, int y, Color c)
             Data[offset++] = _lookup[c.B * BytesPerComponent + 2];
         }
 
-        private static readonly byte[] _lookup = new byte[256 * BytesPerComponent];
+        protected static readonly byte[] _lookup = new byte[256 * BytesPerComponent];
+
+        protected static int BytesPerComponent => 3;
+
+        protected static int BytesPerPixel => BytesPerComponent * 3;
```

Comment:
I'm kinda curious about these two being changed from const to static properties, why not just proetcted const?

Reply:
I'm always having bad feelings about exposing fields. Even if they're protected.

See also [this discussion on StackOverflow](https://stackoverflow.com/a/3182664). I think these are still valid points.

Reply:
We're allowed to make breaking changes later, we assume are derived classed are used only by device implementation which should live in this repo - it's not meant to be a generic purpose abstraction

Reply:
While I do not agree to "ignore" good practice just because we're allowed to make breaking changes, I see the point that this class is internal anyway, so it won't really be a problem to change these back to fields. Will do that later today :-)

Any opinion about renaming the class from BitmapImage to something else? (See my comment [above](https://github.com/dotnet/iot/pull/1800#issuecomment-1052434106)).

Reply:
btw const is not exactly a field, it gets better compiler optimizations